packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -7,72 +7,145 @@
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
 import {
-  getColorV8,
   DEFAULT_THEME,
   retrieveComponentStyles,
   getLineHeight,
-  focusStyles
+  focusStyles,
+  getColor
 } from '@zendeskgarden/react-theming';
 
 import { IGlobalAlertProps } from '../../types';
 
-const COMPONENT_ID = 'notifications.global-alert';
+const COMPONENT_ID = 'notifications.global_alert';
 
 interface IStyledGlobalAlertProps {
-  alertType: IGlobalAlertProps['type'];
+  $alertType: IGlobalAlertProps['type'];
 }
 
+const lightDarkOptions = (lightOffset: number, darkOffset: number) => ({
+  light: { offset: lightOffset },
+  dark: { offset: darkOffset }
+});
```

Comment:
I thought the design directive for `GlobalAlert` was to go with one set of colors, regardless of light/dark mode. Hopefully the color logic below can be simplified.

Reply:
Can you clarify what you are hoping to see / what is aligning for you? I'm not sure how to proceed.

Steph, Lucijan and I had a brief exchange about using variables here instead of the raw colors as the semantic importance of them fits well with GlobalAlert (success, error, warning, etc).